test/e2e/tests/collectibles.spec.js

```diff
@@ -0,0 +1,200 @@
+const { strict: assert } = require('assert');
+const {
+  convertToHexValue,
+  withFixtures,
+  veryLargeDelayMs,
+} = require('../helpers');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Collectibles', function () {
+  const smartContract = SMART_CONTRACTS.COLLECTIBLES;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('should transfer a single NFT from one account to another', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'connected-state',
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver, _, contractRegistry }) => {
+        const contract = contractRegistry.getContractAddress(smartContract);
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // Click transfer
+        await driver.openNewPage(`http://127.0.0.1:8080/?contract=${contract}`);
+        await driver.waitForSelector({
+          css: '#collectiblesStatus',
+          text: 'Deployed',
+        });
+        await driver.delay(veryLargeDelayMs);
```

Comment:
I had to add this delay for the tests to run reliably for me. Without this sometimes clicking on the button doesn't trigger the notification